coverage/src/main/scala/com/databricks/labs/remorph/coverage/ReportEntry.scala

```diff
@@ -14,7 +14,10 @@ case class ReportEntryReport(
     parsing_error: Option[String] = None,
     transpiled: Int = 0, // 1 for success, 0 for failure
     transpiled_statements: Int = 0, // number of statements transpiled
-    transpilation_error: Option[String] = None) {
+    transpilation_error: Option[String] = None,
+    reconciled: Int = 0,
+    reconciled_statements: Int = 0,
+    reconciliation_error: Option[String] = None) {
```

Comment:
```suggestion
    transpilation_error: Option[String] = None) {
```
remove `reconciled` - we don't have cycles to implement it in downstream pipelines

Reply:
OK